src/Audio/Listener.cs

```diff
@@ -69,28 +69,28 @@ public static Vector3f UpVector
         }
         
         #region Imports
-        [DllImport("csfml-audio-2", CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
```

Comment:
maybe, but if you leave it in, it won't compile for .Net Core, that's why my old PR  doesn't have them too #139